.github/workflows/test_suite.yml

```diff
@@ -39,10 +39,23 @@ jobs:
         shell: bash
         run: ${SCCACHE_PATH} --show-stats
 
-      - name: Install rdkit
+      - name: Install dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y build-essential librdkit-dev libssl-dev libboost1.74-dev libboost-serialization1.74-dev pkg-config
+          sudo apt-get install -y build-essential libssl-dev libboost-iostreams-dev libboost-serialization-dev pkg-config curl tar
+
+      - name: Install RDKit
+        run: |
+          cd /tmp
+          if [ $(dpkg --print-architecture) = "amd64" ]; then
+            curl -O https://rdkit-rs-debian.s3.eu-central-1.amazonaws.com/rdkit_2024_03_3_ubuntu_14_04_amd64.tar.gz
+            tar xf rdkit_2024_03_3_ubuntu_14_04_amd64.tar.gz
+          else
+            curl -O https://rdkit-rs-debian.s3.eu-central-1.amazonaws.com/rdkit_2024_03_3_ubuntu_14_04_arm64.tar.gz
+            tar xf rdkit_2024_03_3_ubuntu_14_04_arm64.tar.gz
+          fi
+          cp -r /tmp/rdkit-Release_2024_03_3/Code /usr/local/include/rdkit
+          cp /tmp/rdkit-Release_2024_03_3/build/lib/* /usr/lib/
```

Comment:
use `mv` to make sure there's only one version of these lib/headers in the filesystem

Reply:
Done